src/components/Main/index.js

```diff
@@ -0,0 +1,35 @@
+import styled from "styled-components";
+import Burger from "./Burger";
+function Main(props) {
+  const { prop } = props;
```

Comment:
Give better name for props

```suggestion
  const { price } = props;
```